thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/util/StringUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.linkedin.thirdeye.datalayer.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+
+public class StringUtils {
+  private static Splitter SEMICOLON_SPLITTER = Splitter.on(";").omitEmptyStrings();
+  private static Splitter EQUALS_SPLITTER = Splitter.on("=").omitEmptyStrings();
+
+  private static Joiner SEMICOLON = Joiner.on(";");
+  private static Joiner EQUALS = Joiner.on("=");
+
+  // the following two functions encode and decode properties are moved from class
+  // com.linkedin.thirdeye.controller.mp.function.ThirdEyeAnomalyFunctionUtil
+  public static String encodeCompactedProperties(Properties props) {
```

Comment:
shouldn't this use a standard serializer, like jaxson?

Reply:
I can try using serializer.

Reply:
As the format of the output string is like field1=value1;field2=value2. The jaxson cannot recognize this format. This format is shared with all the other properties in the current design.